src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,88 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTORY = 0.75f;
+    private int capacity = DEFAULT_CAPACITY;
```

Comment:
думаю не стоит присваивать значение константы этой переменной. лучше проинициализировать ее в конструкторе. Если в дальнейшем ты будешь использовать ее как длину массива, то лучше ее вообще убрать, а длину вычислять через table.length. Можно использовать ее как число максимального заполнения перед ресайзом.
```suggestion
    private int capacity;
```